src/cf.data.pre

```diff
@@ -473,6 +473,26 @@ DOC_START
 	CAP_IPC_LOCK capability, or equivalent.
 DOC_END
 
+NAME: hopeless_kid_revival_delay
+COMMENT: time-units
+TYPE: time_t
+LOC: Config.hopelessKidRevivalDelay
+DEFAULT: 60 minutes
+DOC_START
+	Normally, when a kid process dies, Squid immediately restarts the
+	kid. A kid experiencing frequent deaths is marked as "hopeless" for
+	the duration specified by this directive. Hopeless kids are not
+	automatically restarted.
+
+	Currently, zero values are not supported because they result in
+	misconfigured SMP Squid instances running forever, endlessly
+	restarting each dying kid. To effectively disable hopeless kids
+	revival, set the delay to a huge value (e.g., 10 years).
```

Comment:
Since `1 year` does not sound `huge`, the current parsing code supports `10 years` (I presume), and many existing defaults already use plural units, I think `10 years` is acceptable. However, please use `1 year` instead to avoid debating a minor issue.